### DISCUSSION TITLE
お盆

### DISCUSSION THREAD

**Discussion by muro-haya** [muro-haya/yec_robo#2] [General]
お盆期間中のあれこれ相談所

**Comment by eaglraune** [comment]
お盆もお仕事。

**Comment by muro-haya** [comment]
[提出スライド_モデル記載_v5.pptx](https://github.com/user-attachments/files/16487383/_._v5.pptx)
テスト用です。見れるかな？

**Reply by eaglraune** [reply]
みれたよー（^ω^）
